online_meetings/report.py

```diff
@@ -112,6 +132,13 @@ def run_report(api_url: str, json_attribute_name: str,
         total_df.drop_duplicates("uuid", inplace=True)
         logger.info(f"Dataframe with duplicates removed: {len(total_df)}")
 
+    # Create a new calculated column of partipants and duration in the data frame
+    if {'participants', 'duration'}.issubset(total_df.columns):
+        # Fill N/A in participants and duration to zero
+        total_df = total_df.fillna({'participants': 0, 'duration': 0})
+        total_df['particpant_minutes_est'] = total_df.apply(
+            lambda x: round(x['participants'] * to_seconds(x['duration']) / 60, 2), axis=1)
```

Comment:
Do we need to track to the second decimal place?

Reply:
If it's minutes we're calculating, that makes sense to me. Maybe even more precision makes sense, if we're trying to produce an accurate aggregate number.

Reply:
I added the round just because some of these were repeating decimals. However this calculation is an estimate that's and the fraction isn't super relevant. We could go either way on this just round to the minute or just leave it at X digits.